README.md

```diff
@@ -1,239 +1,24 @@
-# benchmarking_representations
+# Benchmarking Representations
 
 Code for training and benchmarking morphology appropriate representation learning methods.
 
-# Preprocessing
-This README gives instructions for running our analysis against preprocessed image data. Because the preprocessing can take a long time, we are publishing preprocessed data along with the original movies. To do the preprocessing yourself, see the instructions in [subpackages/image_preprocessing/README.md](./subpackages/image_preprocessing/README.md).
+Our analysis is organized as follows.
 
-# Installation
+1. Single cell images
+2. Preprocessing (result: pointclouds and SDFs)
+    1. Alignment, masking, and registration
```

Comment:
caveat with preprocessing -  alignment, masking and registration only used for the image models for punctate structures. For polymorphic structures, image models rely on a voxelized SDF/seg image

Reply:
Done in [3f5da87](https://github.com/AllenCell/benchmarking_representations/pull/54/commits/3f5da87097e55887e09f27ec81159bf2edc36d09)